subprojects/docs/src/docs/userguide/installation.adoc

```diff
@@ -15,45 +15,148 @@
 [[installation]]
 == Installing Gradle
 
+You can install the Gradle build tool on Linux, macOS, or Windows.
+This document covers installing using a package manager like SDKMAN!, Homebrew, or Scoop, as well as manual installation.
+
+To upgrade Gradle, use of the <<sec:upgrading_wrapper,Gradle Wrapper>> is the recommended.
+
+You can find all releases and their checksums on the link:https://gradle.org/releases[releases page].
 
 [[sec:prerequisites]]
 === Prerequisites
+Gradle runs on all major operating systems and requires only a link:http://www.oracle.com/technetwork/java/javase/downloads/index.html[Java JDK] version 7 or higher to be installed. To check, run `java -version`. You should see something like this:
```

Comment:
I'd probably say "to run" instead of "to be installed". The currently wording seems to imply that Gradle needs an installer running with Java >= 7 to be installed.

Reply:
Good catch. Adjusted.